cirq-core/cirq/ops/common_channels_test.py

```diff
@@ -244,8 +244,12 @@ def test_deprecated_on_each_for_depolarizing_channel_two_qubits():
     q0, q1 = cirq.LineQubit.range(2)
     op = cirq.DepolarizingChannel(p=0.1, n_qubits=2)
 
-    with pytest.raises(ValueError, match="one qubit"):
-        op.on_each(q0, q1)
+    op.on_each(q0, q1)
```

Comment:
this behavior does not follow the description in the linked comment; namely that multi-qubit oneach should be "strict" and require an iterable of pairs

Reply:
Updated the behavior to only allow a single-level iterable of n-sequences of qubits in the multi-qubit case.

Reply:
I guess "iterable of tuples" means the latter. So I will change that. Is there any desire to support the former too?